.github/workflows/release_please.yml

```diff
@@ -0,0 +1,14 @@
+name: Release please
+
+on:
+  push:
+    branches:
+      - release-please
```

Comment:
So we're not using the main branch as the release branch?

Reply:
Ahh sorry I forgot to revert my test commit. Reverted!